From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Thu, 2 Apr 2015 09:05:12 +0200
Subject: udev: Drop hwdb-update dependency
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

This got introduced in commit d99ce933, but now causes a long dependency chain:

plymouth-start.service → systemd-udevd.service → systemd-hwdb-update.service →
systemd-remount-fs.service → systemd-fsck-root.service → systemd-fsckd.socket

This causes plymouth to start very late, and thus does not cover fsck any more.

We also don't really need this yet; we handle updating hwdb.bin through a dpkg
trigger, and don't support stateless systems in Debian yet.

LP: #1439301
---
 units/systemd-udevd.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-udevd.service.in b/units/systemd-udevd.service.in
index 04f0b98..6e51ce9 100644
--- a/units/systemd-udevd.service.in
+++ b/units/systemd-udevd.service.in
@@ -10,7 +10,7 @@ Description=udev Kernel Device Manager
 Documentation=man:systemd-udevd.service(8) man:udev(7)
 DefaultDependencies=no
 Wants=systemd-udevd-control.socket systemd-udevd-kernel.socket
-After=systemd-udevd-control.socket systemd-udevd-kernel.socket systemd-hwdb-update.service systemd-sysusers.service
+After=systemd-udevd-control.socket systemd-udevd-kernel.socket systemd-sysusers.service
 Before=sysinit.target
 ConditionPathIsReadWrite=/sys
 
